raijin_scripts/test_deaenv_fc/run

```diff
@@ -26,68 +26,49 @@ while [[ $# -gt 0 ]]; do
-    if [[ "$a" == "["* ]]; then
-       configname=$a
-    fi
-
-    if [[ "$a" == "db_database" && "$configname" == "[datacube]" ]]; then
-       databasename="$b"
-    fi
-done < "$CONFIGFILE"
-
 # This is required as FC code expects user to configure work root
 # else work root shall be defaulted to '/g/data/v10/work/' folder
-export DEA_WORK_ROOT="$WORKDIR/output_files/fc"
+export DEA_WORK_ROOT="$WORKDIR/work"
 
-nohup "$SHELL" > "$SUBMISSION_LOG" 2>&1 << EOF &
-echo "Logging results into: ${SUBMISSION_LOG}"
 module use /g/data/v10/public/modules/modulefiles
 module load "${MODULE}"
 
-echo "Database name from config: ${databasename}"
+[[ -z "${DATACUBE_CONFIG_PATH}" ]] && export DATACUBE_CONFIG_PATH="$DC_CONFIG_FILEPATH"
+
+nohup "$SHELL" > "$SUBMISSION_LOG" 2>&1 << EOF &
+echo "Logging results into: ${SUBMISSION_LOG}"
 echo "Loading module ${MODULE}"
 echo ""
 
 # Check if we can connect to the database
-datacube -C "$CONFIGFILE" system check
-
-# Run the initialise Test Database Script
-dea-system -C "$CONFIGFILE" init
-
-# Check if we can connect to the database
-datacube -vv -C "$CONFIGFILE" system check
+datacube -vv -C "$DC_CONFIG_FILEPATH" system check
 
-echo ""
 # Redirect the PBS logs to log_files folder
-cd "$WORKDIR/log_files" || exit 0
+cd "$WORKDIR" || exit 0
 
+echo ""
 echo "Applying fractional cover on ls7_fc_albers product for the year ${YEAR}" with tag "${TAG}"
 echo "------------------------------------------------------------------------------------------"
-datacube-fc submit --app-config "$WORKDIR"/fc_configfiles/ls7_fc_albers.yaml --project "${PROJECT}" --queue "${QUEUE}" -C "$CONFIGFILE" -vvv --year "${YEAR}" --tag "${TAG}"
+datacube-fc submit --app-config "$WORKDIR"/fc_configfiles/ls7_fc_albers.yaml --project "${PROJECT}" --queue "${QUEUE}" -C "$DC_CONFIG_FILEPATH" -vvv --year "${YEAR}" --tag "${TAG}"
 
 echo "Applying fractional cover on ls8_fc_albers product for the year ${YEAR}" with tag "${TAG}"
 echo "------------------------------------------------------------------------------------------"
-datacube-fc submit --app-config "$WORKDIR"/fc_configfiles/ls8_fc_albers.yaml --project "${PROJECT}" --queue "${QUEUE}" -C "$CONFIGFILE" -vvv --year "${YEAR}" --tag "${TAG}"
+datacube-fc submit --app-config "$WORKDIR"/fc_configfiles/ls8_fc_albers.yaml --project "${PROJECT}" --queue "${QUEUE}" -C "$DC_CONFIG_FILEPATH" -vvv --year "${YEAR}" --tag "${TAG}"
 
 echo "Applying fractional cover on ls5_fc_albers product for the year ${YEAR}" with tag "${TAG}"
 echo "------------------------------------------------------------------------------------------"
-datacube-fc submit --app-config "$WORKDIR"/fc_configfiles/ls5_fc_albers.yaml --project "${PROJECT}" --queue "${QUEUE}" -C "$CONFIGFILE" -vvv --year "${YEAR}" --tag "${TAG}"
+datacube-fc submit --app-config "$WORKDIR"/fc_configfiles/ls5_fc_albers.yaml --project "${PROJECT}" --queue "${QUEUE}" -C "$DC_CONFIG_FILEPATH" -vvv --year "${YEAR}" --tag "${TAG}"
```

Comment:
We don't need to be rerunning FC for landsat 5 all the time since there isn't new data arriving any more.

It's not a big problem, but it is a sign we may need to rethink how we're scheduling these sorts of regular tasks.